.circleci/config.yml

```
@@ -29,15 +29,20 @@ jobs:
       - run:
           name: Build Docker Image if Necessary
           command: |
-            IMAGE_TAG="$(git rev-list -n1 HEAD -- Dockerfile package-lock.json)"
-            echo "Using $IMAGE_TAG"
             if  [ "$CIRCLE_REPOSITORY_URL" != "git@github.com:hashicorp/terraform-website.git" ]; then
               echo "Not Terraform Website Repo, not building website docker image"
```

maybe this would be nice

```suggestion
              echo "Not Terraform Website Repo, not building website docker image"
              exit 0
```